duckduckgo/iOS
# Update app to allow sending of anonymous feedback

Task/Issue URL: https://app.asana.com/0/72649045549333/675946344778515

**Description**:
Replaces the feedback email with a feedback url to allow users to send anonymous feedback.

**Steps to test this PR**:
1. Go to app settings
1. Tap the "Send Feedback" option and note that a "Feedback" page opens in a new tab
1. Enter details and submit the form

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
